Readme.md

```diff
@@ -148,6 +166,41 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Collaborators
+
+* [Jon Church](https://github.com/jonchurch) -
+  **Jon Church** <<me@jonchurch.com>>
+* [Wes Todd](https://github.com/wesleytodd) -
+  **Wes Todd** <<wes@wesleytodd.com>>
```

Comment:
We can remove these. That group is one I made and added @jonchurch but it was an administrative reason not really something we should keep.